pkg/registry/apps/rest/storage_apps.go

```diff
@@ -31,25 +31,27 @@ import (
 	statefulsetstore "k8s.io/kubernetes/pkg/registry/apps/statefulset/storage"
 )
 
-type RESTStorageProvider struct{}
+// StorageProvider is a struct for apps REST storage.
+type StorageProvider struct{}
```

Comment:
Just curious: Did the old name cause a linter issue?

Reply:
Ah, I see. It's because of the stutter in `rest.RESTStorageProvider`.

Reply:
yep! Exact message: `pkg/registry/apps/rest/storage_apps.go:34:6: type name will be used as rest.RESTStorageProvider by other packages, and that stutters; consider calling this StorageProvider`. Thanks!